tests/tracking/test_tracking.py

```diff
@@ -9,7 +9,7 @@
 
 import mlflow
 from mlflow import tracking
-from mlflow.entities import Experiment, RunStatus
+from mlflow.entities import Experiment, RunStatus, LifecycleStage
```

Comment:
Nit: `Experiment` is no longer used and should be removed (though my local linter isn't catching this)